CONTRIBUTING.md

```diff
@@ -80,3 +80,18 @@ Each PR will not be able to have more than one change file. (e.g. A bug fix and
 ### Q. I made new changes during my PR after I already generated the change file/I made a mistake while generating the change file! How can I edit it?
 
 You can manually edit the change file- it's actually just a JSON file. Locate your change file under the [change](./change) directory. If you're having trouble finding your change file, try searching the directory in your IDE for your change description or going through your branch's commit history for the change file you had committed.
+
+## Contribution Guide for Users without Write Access
```

Comment:
Should this be updated with guidance around how to join [Microsoft](https://repos.opensource.microsoft.com/)/[OfficeDev](https://repos.opensource.microsoft.com/orgs/OfficeDev/join)/[Teams Client SDK Contributors](https://github.com/orgs/OfficeDev/teams/teams-client-sdk-contributors)?

That will only work for Microsoft Employees, but so far I've had a few different people reach out and ask how to get access to make PRs / update comments etc. and then I have to point them to those instructions.